File: routes/api/userdata.js
const express = require('express');
const router = express.Router();

const UserData = require('../../models/UserData');

// @route  GET api/userdata
// @desc   Get listing of userdata
// @access Public

router.get('/', (req, res) => {
    UserData
        .find()
        .then(UserData => res.json(UserData));
});

// @route  POST api/userdata
// @desc   Create an userdata
// @access Public

router.post('/', (req, res) => {
    const newUserData = new UserData({
        name: req.body.name,
        nationality: req.body.nationality,
        nationality_pt: req.body.nationality_pt,
        maritalStatus: req.body.maritalStatus,
        maritalStatus_pt: req.body.maritalStatus_pt,
        yearOfBirth: req.body.yearOfBirth,
        address: req.body.address,
        city: req.body.city,
        state: req.body.state,
        homePhone: req.body.homePhone,
        cellPhone: req.body.cellPhone,
        email: req.body.email,
        website: req.body.website,
    });
    newUserData.save().then(UserData => res.json(UserData));
});

module.exports = router;

File: dbHelper/classes/userData.py
from classes.collection import Collection

from classes.helpers.functions import (
    presentChoiceString,
    waitForInput,
    handleBool,
    handleYear
)

class UserData(Collection):

    def __init__(self, name, workName, database):
        super().__init__(name, workName, database)
    
    def insertDocument(self):
        print("\n* ADDING NEW USER DATA:")
        print(" * name:")
        name = presentChoiceString()
        print(" * nationality:")
        nationality = presentChoiceString()
        print(" * nationality (Portuguese):")
        nationality_pt = presentChoiceString()
        print(" * marital status:")
        maritalStatus = presentChoiceString()
        print(" * marital status (Portuguese):")
        maritalStatus_pt = presentChoiceString()
        print(" * year of birth:")
        yearOfBirth = handleYear()
        print(" * address:")
        address = presentChoiceString()
        print(" * city:")
        city = presentChoiceString()
        print(" * state:")
        state = presentChoiceString()
        print(" * home phone:")
        homePhone = presentChoiceString()
        print(" * cell phone:")
        cellPhone = presentChoiceString()
        print(" * email:")
        email = presentChoiceString()
        print(" * website:")
        website = presentChoiceString()

        newUserData = {
            "name" : name,
            "nationality" : nationality,
            "nationality_pt" : nationality_pt,
            "maritalStatus" : maritalStatus,
            "maritalStatus_pt" : maritalStatus_pt,
            "yearOfBirth" : yearOfBirth,
            "address" : address,
            "city" : city,
            "state" : state,
            "homePhone" : homePhone,
            "cellPhone" : cellPhone,
            "email" : email,
            "website" : website,
        }
        self.database[self.workName].insert_one(newUserData)
        print("\n* NEW USER DATA ADDED!")

    def modifyDocument(self):
        document = self.findDocument()

        if document == None:
            return

        print("\n* MODIFYING USER DATA:")
        for key in list(document.keys()):
            if (key != "_id" and key != "__v"):
                print("* Change '{}' field? (Y/N)".format(key))
                choice = handleBool()
                if choice:
                    print("* New '{}':".format(key))
                    value = presentChoiceString()

                    self.database[self.workName].find_one_and_update(
                        { "_id" : document["_id"] },
                        { '$set' : { key : value } }
                    )
        print("\n* DOCUMENT UPDATED!")
        waitForInput(True)